web/app/themes/clarity/inc/admin/prior-party/prior-party-banner-admin.php

```diff
@@ -212,6 +212,19 @@ public function page(): void
             $stop = new \DateTime($this->banner["end_date"]);
 
 
+            // Init an array to hold the query string for viewing the banner.
+            $link_view_queries = [];
+            // If the banner is inactive, we need to set a query string to show it.
+            if(!$this->banner['banner_active']) {
+                $link_view_queries['preview_unpublished'] = '';
```

Comment:
Brilliant result for the name 😄

Reply:
Cheers, thanks for the suggestion to update it.